File: Iniciante/URI 1007.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
var c = parseInt(lines.shift());
var d = parseInt(lines.shift());
var diferenca = a * b - c * d;
   
console.log('DIFERENCA = ' + diferenca);

File: Iniciante/URI 1007.r
input = file('stdin', 'r')

a = as.double(readLines(input, n=1))
b = as.double(readLines(input, n=1))
c = as.double(readLines(input, n=1))
d = as.double(readLines(input, n=1))
diferenca = a * b - c * d

write(sprintf("DIFERENCA = %d", diferenca), '')